extensions/aerospace/assets/scripts/list-windows.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# References:
+# - https://nikitabobko.github.io/AeroSpace/commands#list-windows
+# - https://www.alfredapp.com/help/workflows/inputs/script-filter/json/
+
+result=$(aerospace list-windows --json --workspace visible --format "%{app-name} %{window-title} %{window-id} %{app-pid} %{workspace} %{app-bundle-id}")
```

Comment:
`--workspace visible` will show windows on all visible workspaces, not only the focused workspace (This difference is observable only in multi monitor setup). But according to your screenshot, you want to show workspaces on the focused workspace

If so, you should use `--workspace focused`

Reply:
Fixed